lib/web3/event.js

```diff
@@ -30,7 +30,8 @@ var watches = require('./methods/watches');
 /**
  * This prototype should be used to create event filters
  */
-var SolidityEvent = function (json, address) {
+var SolidityEvent = function (web3, json, address) {
```

Comment:
These events maybe solidly specific, but we have to check that with christian. Otherwise they should also be renamed to something more general.